package-lock.json

```diff
@@ -1627,6 +1627,11 @@
           "requires": {
             "ms": "2.0.0"
           }
+        },
```

Comment:
I am not sure why `package-lock.json` file needs to be updated when `package.json` doesn't change.

Reply:
Whoops.  I installed qs briefly, then didn't need it. Will revert this.